mteb/tasks/Classification/ita/ItaHateClassification.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ItaHateClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="ItaHateClassification",
+        dataset={
+            "path": "Paul/hatecheck-italian",
+            "revision": "21e3d5c827cb60619a89988b24979850a7af85a5",
+        },
+        description="""Hate speech detection dataset with binary
+                       (hateful vs non-hateful) labels. Includes 25+ distinct types of hate
+                       and challenging non-hate. Multilingual datase released as 10 unilingual models
+                     """,
+        reference="https://aclanthology.org/2022.woah-1.15/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ita-Latn"],
+        main_score="accuracy",
+        date=("2021-11-01", "2022-02-28"),
+        form=["written"],
+        domains=None,
+        task_subtypes=["Sentiment/Hate speech"],
+        license="cc-by-4.0",
+        socioeconomic_status=None,
+        annotations_creators="expert-annotated",
+        dialect=None,
```

Comment:
```suggestion
        dialect=[],
```
basically saying I have considered if there are meaningful dialects to annotate and there is none. As opposed to None which is unannotated.